test/controllers/images_controller_test.rb

```diff
@@ -12,8 +12,9 @@ def setup
     system('cp test/fixtures/demo.png public/system/images/1/original/')
     system('mkdir -p public/warps/saugus-landfill-incinerator')
 
-    file_location = Rails.root + 'test/fixtures/demo.png'
-    @uploaded_data = Rack::Test::UploadedFile.new(file_location.to_s, 'demo.png', "image/png")
+    @file ||= File.open(File.expand_path(Rails.root + 'test/fixtures/demo.png', __FILE__))
+    @uploaded_data = ActionDispatch::Http::UploadedFile.new(tempfile: @file, filename: File.basename(@file), type: "image/png")
+    PaperTrail.enabled = true
   end
```

Comment:
This change is causing the controller tests to fail
@alaxalves

Reply:
I don't understand how it's working in main but not here

Reply:
Maybe some new Rails 5 class related thing